### DISCUSSION TITLE
energy management problem

### DISCUSSION THREAD

**Discussion by kevin031060** [XiongPengNUS/rsome#18] [Q&A]
I have a energy management problem with solar power uncertainty. However, the random variable of solar power is not [a+delta_i,a-delta_i]. It is [a+x, a-y], where x and y are different values for each hour. Is it the right way to transform [a+x, a-y] to the form of [a+delta_i,a-delta_i]? 

I'm new to robust optimization. Thanks for who can help me.

**Comment by chenzjames** [comment]
You can try to look at the paper "Price of Robustness". It does not matter in terms of deterministic reformulation whether the uncertainty set is symmetric or not.
